packages/dynamodb/index.js

```diff
@@ -0,0 +1,620 @@
+import { createHash, randomBytes } from "crypto";
+
+export default function Adapter(config, options = {}) {
+  if (!config.AWS) {
+    console.error("CONFIG_ADAPTER", "AWS is not defined in adapter config");
+    return Promise.reject(new Error("CONFIG_ADAPTER"));
+  }
+
+  if (!config.tableName) {
+    console.error(
+      "CONFIG_ADAPTER",
+      "tableName is not defined in adapter config"
+    );
+    return Promise.reject(new Error("CONFIG_ADAPTER"));
+  }
+
+  const TableName = config.tableName;
+  const DynamoClient = new config.AWS.DynamoDB.DocumentClient();
+
+  async function getAdapter(appOptions) {
+    // Display debug output if debug option enabled
+    function _debug(...args) {
+      if (appOptions.debug) {
+        console.log("[next-auth][debug]", ...args);
+      }
+    }
+
+    const defaultSessionMaxAge = 30 * 24 * 60 * 60 * 1000;
+    const sessionMaxAge =
+      appOptions && appOptions.session && appOptions.session.maxAge
+        ? appOptions.session.maxAge * 1000
+        : defaultSessionMaxAge;
+    const sessionUpdateAge =
+      appOptions && appOptions.session && appOptions.session.updateAge
+        ? appOptions.session.updateAge * 1000
+        : 0;
+
+    async function createUser(profile) {
+      _debug("createUser", profile);
+
+      const userId = randomBytes(16).toString("hex");
+      const now = new Date();
+      let item = {
+        pk: `USER#${userId}`,
+        sk: `USER#${userId}`,
+        id: userId,
+        type: "USER",
+        name: profile.name,
+        email: profile.email,
+        iamge: profile.image,
```

Comment:
I believe is just a minor typo, `iamge` should be `image`

Reply:
Indeed, thanks :)